ape_trezor/hdpath.py

```diff
@@ -0,0 +1,33 @@
+class HDPath:
+    """
+    A class representing an HD path. This class is the base class
+    for both account specific HD paths (:class:`~ape_trezor.hdpath.HDPath`)
+    as well as the derivation HD path class :class:`~ape_trezor.hdpath.HDBasePath`.
+    """
+
+    def __init__(self, path: str):
+        path = path.rstrip("/")
+        if not path.startswith("m/"):
+            raise ValueError("HD path must begin with m/")
+
+        self.path = path
+
+    def __str__(self):
+        return self.path
+
+
+class HDBasePath(HDPath):
+    """
+    A derivation HD path useful for creating objects of type
+    :class:`~ape_trezor.hdpath.HDPath`.
+    """
+
+    DEFAULT = "m/44'/60'/0'/0"
+
+    def __init__(self, base_path=DEFAULT):
```

Comment:
should this be `None`? wasn't aware of how this works when using a class var as a default arg

Reply:
There is another often overlooked benefit to defaulting to `None` and setting the actual default to the `str` within the method... The benefit is that when a user dynamically passes in `None` (perhaps via some scripting, not ad-hoc), we can still use the real default `str`. Otherwise, `None` overwrites the default.

Reply:
Yes! This is a nasty little bug in the making :sweat_smile:

Reply:
```suggestion
    def __init__(self, base_path=None):
```

Reply:
I updated it in the ledger repo as well. I'll go ahead and merge the changes today.